js/index.js

```diff
@@ -1,7 +1,53 @@
 // Iteration 1: Names and Input
 
+const hacker1 = 'Lucas';
+
+console.log(`"The driver´s name is ${hacker1}"`);
+
+const hacker2 = 'Jesus';
+
+console.log(`"The navigator´s name is ${hacker2}"`)
 
 // Iteration 2: Conditionals
 
+let driverNameLength = hacker1.length;
+
+let navigatorNameLength = hacker2.length;
+
+if (driverNameLength > navigatorNameLength){
+  console.log(`"The driver has the longest name, it has ${driverNameLength} characters"`)
+} else if (driverNameLength < navigatorNameLength) {
+  console.log(`"It seems that the navigator has the longest name, it has ${navigatorNameLength} characters"`)
+} else {
+  console.log(`"Wow, you both have equally long names, ${driverNameLength} characters!"`)
+}
+
+// Iteration 3: Loops
+
+let driversCharacters = "";
+
+for (let i = 0; i < hacker1.length; i++) {
+  driversCharacters += hacker1[i] + " ";
+}
+
+console.log(driversCharacters.toUpperCase());
```

Comment:
great.. you have space at the end of the name, you can use .slice() or .trim() to remove it.